addons/website_rating/views/rating_templates.xml

```diff
@@ -1,5 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+    <!--
+        Rating widget static: show 5 stars (full or empty) regarding the given rating_avg and rating_count
+    -->
+    <template id="rating_widget_stars_static" name="Rating: static star widget">
+        <t t-set="rating_avg" t-value="round(rating_avg * 100) / 100 / 2"/>
+        <t t-set="val_decimal" t-value="round(rating_avg % 1, 1)"/>
+        <t t-set="val_integer" t-value="int(rating_avg)"/>
+        <t t-set="empty_star" t-value="5 - (val_integer+1) if val_decimal else 5 - val_integer"/>
+        <div class="o_website_rating_static" t-att-style="'display:inline' if inline_mode else ''" t-att-title="rating_avg">
```

Comment:
inline_mode and 'display:inline' 


```python
t-att-style='None' => no style tag after rendering
t-att-style='""' => 'style' tag empty after rendering
```

And we prefer syntax btw with condition before, if the else have no impact... 

```python
x = 0
condition = True
a = condition and x or 'False' ==> return "False"
while a = x if condition or 'False' ==> return 0 (x)
```